spot-client/src/spot-remote/ui/components/dial-pad/StatelessDialPad.js

```diff
@@ -136,10 +137,12 @@ export default class StatelessDialPad extends React.Component {
                         <div className = 'dial-pad-footer-button'>
                             <button
                                 className = { callButtonClassName }
-                                disabled = { this.props.disableCallButton }
+                                disabled = { this.props.disableCallButton || this.props.dialingInProgress }
```

Comment:
Shouldn't disableCallButton be set to true instead of also relying on dialingInProgress? Seems like a redundant prop.

Reply:
There are two states "disabled call button" and "loading icon on call button".

Reply:
maybe I can add extra variable on the top which will control the button's disabled state? would that make things any better?